ui/package.json

```diff
@@ -225,7 +223,6 @@
     "qs": "^6.3.0",
     "serialize-javascript": "^3.1.0",
     "underscore": "^1.12.1",
-    "trim": "^0.0.3",
     "xmlhttprequest-ssl": "^1.6.2",
```

Comment:
This is the only resolution I left that I couldn't find when searching the codebase as it was added recently when upgrading to Ember 4.12 https://github.com/hashicorp/vault/pull/22122